src/stp/plays/contested/GetBallPossession.cpp

```diff
@@ -16,15 +13,15 @@
 namespace rtt::ai::stp::play {
 
 GetBallPossession::GetBallPossession() : Play() {
-    startPlayInvariants.clear();
-    startPlayInvariants.emplace_back(std::make_unique<invariant::NormalPlayGameStateInvariant>());
-    startPlayInvariants.emplace_back(std::make_unique<invariant::BallIsFreeInvariant>());
-    startPlayInvariants.emplace_back(std::make_unique<invariant::BallClosestToUsInvariant>());
+    startPlayEvaluation.clear();
+    startPlayEvaluation.emplace_back(GlobalEvaluation::NormalPlayGameState);
+//    startPlayEvaluation.emplace_back(GlobalEvaluation::BallIsFree);
+//    startPlayEvaluation.emplace_back(GlobalEvaluation::BallClosestToUs);
```

Comment:
Why is this commented? You need to be kind to future readers of the code, otherwise they're just going to remove it.

Reply:
Next step was to implement plays (so reimplement these evaluations)